### ISSUE TITLE
Not working with JAX 0.4

### ISSUE THREAD

**Issue by YanniPapandreou** [wesselb/mlkernels#4]
Hi, 

Firstly thanks for creating this package. I have been finding it very useful in my research.

I recently updated JAX to version 0.4.1. The JAX backend for mlkernels no longer seems to work. Running the following code yields the following error:

Code:

```python
from mlkernels.jax import EQ
import jax.numpy as np

x = np.array([1.0,2.0])

k = EQ()

k(x,x)
```

Error:
```
RuntimeError: For kernel "EQ()", could not resolve arguments "[1. 2.]" and "[1. 2.]".
```

Would it be possible to support the newer version of JAX? I am happy to help with this but unsure of where to start... 

I did notice that arrays in JAX now seem to have type `jaxlib.xla_extension.Array` instead of `jaxlib.xla_extension.DeviceArray`. I am not sure how to add this type to `mlkernels`. Any help would be appreciated :)

**Comment by YanniPapandreou** [comment]
Actually I think I managed to fix this by changing a line in your package [lab](https://github.com/wesselb/lab) in the `types.py` file from:

```python
_jax_tensor = ModuleType("jax.interpreters.xla", "DeviceArray")
```

to

```python
_jax_tensor = ModuleType("jax", "Array")
```

This is described in a JAX migration guide [here](https://jax.readthedocs.io/en/latest/jax_array_migration.html). I am not sure if this breaks anything else. Happy to open an issue on [lab](https://github.com/wesselb/lab) if you want!

**Comment by wesselb** [comment]
Hey @YanniPapandreou!

@stoprightthere kindly provided a PR which I believe should fix this. I have just released `v1.4.32` of LAB which incorporates this PR. Would you be able to see if `pip install backends==1.4.32` fixes this issue for you?

**Comment by YanniPapandreou** [comment]
Hi @wesselb, updating `backends` to `v1.4.32` does indeed fix the problem! Thanks a lot :)
